example/spec/parallel/testAPI/snsRuleSpec.js

```diff
@@ -253,19 +276,20 @@ describe('The SNS-type rule', () => {
     });
 
     it('saves the new rule.value', () => {
+      if (beforeAllFailed) fail(beforeAllFailed);
       expect(putRule.rule.value).toEqual(newTopicArn);
     });
 
     it('deletes the old subscription', async () => {
+      if (beforeAllFailed) fail(beforeAllFailed);
       expect(await getNumberOfTopicSubscriptions(snsTopicArn)).toBe(0);
     });
 
     it('adds the new policy and subscription', async () => {
+      if (beforeAllFailed) fail(beforeAllFailed);
       const { Policy } = await lambda().getPolicy({ FunctionName: consumerName }).promise();
       const { Statement } = JSON.parse(Policy);
-      expect(Statement.length).toEqual(1);
-      expect(Statement[0].Sid).toEqual(`${ruleName}Permission`);
-      expect(await getNumberOfTopicSubscriptions(newTopicArn)).toBe(1);
+      expect(Statement.some((s) => s.Sid === expectedStatementId));
```

Comment:
```suggestion
      expect(Statement.some((s) => s.Sid === expectedStatementId)).toBeTrue();
```

I think we want to make an assertion here

Reply:
Ah, yes.